.github/workflows/main.yml

```diff
@@ -137,3 +137,48 @@ jobs:
       with:
         working-directory: examples/basic
         args: --tests
+
+  # run integration tests with our main consumers
+  integration:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        repo: ["aws/s2n-quic"]
+
+    steps:
+    - uses: actions/checkout@v3.5.0
+
+    - uses: actions/checkout@v3.5.0
+      with:
+        repository: ${{ matrix.repo }}
+        path: "target/integration/${{ matrix.repo }}"
+
+    - name: Add patch
+      working-directory: target/integration/${{ matrix.repo }}
+      run: |
+        cat <<EOF >> Cargo.toml
+        [patch.crates-io]
+        bolero = { path = "../../../../bolero" }
+        bolero-engine = { path = "../../../../bolero-engine" }
+        bolero-generator = { path = "../../../../bolero-generator" }
+        bolero-generator-derive = { path = "../../../../bolero-generator-derive" }
```

Comment:
Not a priority, but we could have a script that does this patching in the future.

Reply:
Yeah I can open an issue for it and link it